js/util.js

```diff
@@ -90,3 +90,33 @@ util.guessLineEndings = function(text) {
 
   return (hasCRLF ? '\r\n' : '\n');
 };
+
+/**
+ * @param {?string} opt_content Optional content.
+ * Creates a unified session that can be read from any supported editor
+ */
+util.createUnifiedSession = function(opt_content) {
+  const textarea = document.createElement('textarea');
+  textarea.value = opt_content || '';
+  return {
+    codemirror: new CodeMirror.Doc(opt_content || ''),
+    textarea
+  }
+}
+
+/**
+ * @param {EditSession} session
+ * @param {string} updated Which text source is the source of truth.
+ * @param {string} lineEndings What to use as a line ending
+ * Syncs the multiple formats of a unified session.
```

Comment:
I think this description could be clearer - this is specifically to maintain the undo/redo stack right?

Reply:
Yeah sorta, it's so that on any edit both objects are updated for a seemless experience swithing back and forth in and out of a11y mode. Will update the comment

Reply:
Ack.